scipy/stats/tests/test_stats.py

```diff
@@ -3243,6 +3244,23 @@ def test_result_attributes(self):
         attributes = ('statistic', 'pvalue')
         check_named_results(res, attributes)
 
+    def test_nist(self):
+        # The npz file with the nist test cases data is assumed to be in the
+        # same directory as this file.
+        nist = np.load(os.path.abspath(os.path.join(os.path.dirname(__file__),
+                                                    'nist_anova.npz')))
+        rtol = 1e-7
+        for test_case in nist:
+            (y, x, f, dfbn, dfwn, R2,
+             resstd, certified, caty) = nist[test_case]
```

Comment:
The indentation (single space) here is a bit odd. If you move ``resstd`` one line up and then align ``)`` with the end of the line above again it'll look better.